app/tests/store.test.js

```diff
@@ -6,28 +6,9 @@ import { browserHistory } from 'react-router-dom';
 import configureStore from '../configureStore';
 
 describe('configureStore', () => {
-  let store;
-
-  beforeAll(() => {
-    store = configureStore({}, browserHistory);
-  });
-
-  describe('injectedReducers', () => {
-    it('should contain an object for reducers', () => {
-      expect(typeof store.injectedReducers).toBe('object');
-    });
-  });
-
-  describe('injectedSagas', () => {
-    it('should contain an object for sagas', () => {
-      expect(typeof store.injectedSagas).toBe('object');
-    });
-  });
-
-  describe('runSaga', () => {
-    it('should contain a hook for `sagaMiddleware.run`', () => {
-      expect(typeof store.runSaga).toBe('function');
-    });
+  it('should return a redux store', () => {
+    const store = configureStore({}, browserHistory);
```

Comment:
Here browserHistory is undefined because it's an old obsolete API from react-router. You'll see it's been removed in #2791. You'll get a merge conflict here. You can just remove that argument and the test should pass just fine without it.